aips/create_dip.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+"""
+Create DIP from AIP
+
+Downloads an AIP from the Storage Service and creates a DIP
+"""
+
+import argparse
+import logging
```

Comment:
I thought it could be automated in the future so I copied the logging system from the transfers script. I think the sophistication of it shouldn't be a big issue now that it's there.